.gitignore

```diff
@@ -6,32 +6,18 @@ lerna-debug.log
 
 coverage
 .DS_Store
-dist
-build/
-website/.cache
 tsconfig.tsbuildinfo
 
 .idea/
 
 .cache-test
-/build-dist/
-
-*.js
 
-!jest-*.js
-!dist-prepare.js
+/build/
+/build-dist/
 
 /mitm/socket/connect
-!mitm/install.js
 /mitm/socket/build/
-/.mitm-ca
-/**/.mitm-ca
 
 emulators/data/DOM-*.json
-!/emulator-plugins/shared/injected-scripts/*.js
-!/emulator-plugins/shared/test/inspectHierarchy.js
-/emulator-plugins/shared/injected-scripts/index.js
-
 
-/.sessions
```

Comment:
Shouldn't this also still be .gitignored?

Reply:
Yep.